### ISSUE TITLE
Data submission to ENCODE: cs-386

### ISSUE THREAD

**Issue by twang15** [StanfordBioinformatics/pulsar_lims#133]
Submission begins

**Comment by twang15** [comment]
Submission with two issues: https://www.encodeproject.org/experiments/ENCSR942XQI/
1. Missing nih_institutional_certification
2. Missing biosample characterization

**Comment by twang15** [comment]
> Submission with two issues: https://www.encodeproject.org/experiments/ENCSR942XQI/
> 
> 1. Missing nih_institutional_certification
> 2. Missing biosample characterization

Hi Annika and Lixia,

CS-386 has been submitted but there are two issues:
1.	Missing NIH certificate
2.	Missing biosample characterizations

Please help me fill this information in pulsar for a complete submission on ENCODE.
Thanks a lot!

Best,
Tao

**Comment by twang15** [comment]
Hi Tao, I added the NIH cert. number (NIC00007).

I have to send an email to Cory for the validations.

Thanks, Annika

**Comment by twang15** [comment]
Hi Cory,

Do you have biosample validations for the RAD21 in K562 that you sent us for cut and run?

Thanks, Annika

**Comment by twang15** [comment]
> Hi Cory,
> 
> Do you have biosample validations for the RAD21 in K562 that you sent us for cut and run?
> 
> Thanks, Annika

I'm out of the lab until Thursday.  I believe RAD21 was deemed a non TF so I can't be certain that I have complete validation.  It was an early factor so I may only have an IP.

**Comment by twang15** [comment]
Hi Cory,
 
Please submit the IP into Pulsar and I will continue the submission for cs-386.
 
Thanks,
Tao

**Comment by twang15** [comment]
> Hi Cory,
> 
> Please submit the IP into Pulsar and I will continue the submission for cs-386.
> 
> Thanks,
> Tao

B-10302 has the IP and PCR.  Those didn't carry over to the replicates, I don't know why.  RAD21 was an early sample so only the pool was IP'd, after that they were split into replicates.

**Comment by twang15** [comment]
Hi Cory,
 
B-10302 is for cs-86, but we are talking about cs-386.
Could you double-check them in pulsar?
 
Thanks,
Tao

**Comment by twang15** [comment]
Hi Tao,

It’s both RAD21. Cory grew the cells again (in parallel with other cells for better comparison of the cut and run assay for Anshul) and we repeated the ChIP. So you can take the same IP from the old CS because it’s a ’technical’ replicate.

Let me know if that makes sense, best, Annika

**Comment by twang15** [comment]
Hi Annika,

Thanks for the explanations.
I will take a note for later submission.

Best,
Tao

**Comment by twang15** [comment]
submit IP for 12980 and 12981 from cs-86's biosample parents.

**Comment by twang15** [comment]
To be reviewed: Missing compliant biosample characterization

**Comment by twang15** [comment]
### Reopen because of Review results:
Not compliant biosample characterization

**Comment by twang15** [comment]
Hi Annika,

The IPs of cs-386  https://www.encodeproject.org/experiments/ENCSR942XQI/ are not compliant.
How shall we proceed to fix it?

Best,
Tao

**Comment by twang15** [comment]
Hi Tao,
 
I found a problem with this experiment: https://www.encodeproject.org/experiments/ENCSR942XQI/ 
 
Those two libraries ENCLB961DOA and ENCLB215FJK are associated with antibody https://www.encodeproject.org/antibodies/ENCAB728YTO/ which is GFP. But they are processed with IgG antibody.  It is correct in Pulsar. Could you check what went wrong here?
 
Thanks, Annika

**Comment by twang15** [comment]
Hi Annika,
 
A possible explanation is:
ENCLB961DOA and ENCLB215FJK reused biosample ENCBS963OUI and ENCBS989NSH, respectively.
 
What matters more is how to solve this issue:
1. register the IgG antibody. Should it be done by DCC or us?
2. correct the IgG for ENCLB961DOA and ENCLB215FJK
 
Thanks,
Tao

**Comment by twang15** [comment]
Ha! You’re right. It’s not on the portal. Ok, change of plan: let’s remove the libraries from the portal that use the IgG antibody (ENCLB961DOA and ENCLB215FJK)
 and only leave the ones with GFP. Since the AB is not registered yet (and not through the review process) it takes longer than anticipated. The DCC can process the ChIP libraries anyway and they don’t need the IgG libraries. 

Thanks, Annika

**Comment by twang15** [comment]
Remove rep1 technical rep 2 /replicates/03b5a761-b895-498e-b541-dca6927e5e04/
Remove rep2 technical rep2 /replicates/a1d9c043-9570-4fe6-95f0-1e8b31c807d6/

**Comment by twang15** [comment]
Hi Jessica,

We would like to remove the following two libraries from this chip-seq https://www.encodeproject.org/experiments/ENCSR942XQI/


ENCLB961DOA
ENCLB215FJK

I managed to unlink their replicates from the experiment, but need your help to unlink the libraries.

Could you please help me with this?

Thanks a lot!

Best,
Tao

**Comment by twang15** [comment]
Hi Ingrid and Jennifer, 

Re ENCSR942XQI 
The two libraries that had to be deleted were ENCLB961DOA and ENCLB215FJK. And I cannot see them on the portal anymore so I guess this is already done. The other ones, we should keep.

Thanks, Annika

**Comment by twang15** [comment]
Hi Annika,

It looks like those two libraries are "in progress" but are not attached to any replicate or dataset. We can delete them if they will no longer be used anywhere.

Best,
Jennifer

**Comment by twang15** [comment]
Yes, that works; it’s only ENCLB961DOA and ENCLB215FJK. Thanks, Annika

**Comment by twang15** [comment]
Waiting for DCC to delete fastq files for
ENCLB961DOA and ENCLB215FJK
